NosTests/Models/NoteParserTests.swift

```diff
@@ -233,20 +236,18 @@ final class NoteParserTests: CoreDataTestCase {
         let content = "check this \(event). Bye!"
         let tags: [[String]] = [[]]
 
-        let expectedContent = "check this 🔗 Link to note. Bye!"
-        let (attributedContent, _) = sut.parse(
+        let expectedContent = "check this . Bye!"
```

Comment:
I wasn't sure what to do with this. It's probably a Product question for Linda what to do when a note link is inline with other content as opposed to just at the end of a note.

Reply:
Yeah, I think it's fine to just leave like this for now, and maybe ask Linda's thoughts.